src/kvilib/irc/KviControlCodes.cpp

```diff
@@ -306,4 +306,102 @@ namespace KviControlCodes
 		}
 	}
 #endif
+
+	// Get extended (16-98) mIRC color.
+	// Unlike the 0-15 ones, these are not configurable.
+	// https://modern.ircdocs.horse/formatting.html#colors-16-98
+	// Returns (kvi_u32_t)-1 if index is out of bounds.
+	kvi_u32_t getExtendedColor(int index)
```

Comment:
Could we just extend the original color array and include them all in one place?  Seems like wasted operations to have to determine if it's above or below 16 with special indexing to account for missing the first 16 values.

Reply:
The original color array is an array of `KviColorOption`. That doesn't look like an appropriate place for hard-coded, immutable color values.